src/sass/partials/_nav.scss

```diff
@@ -25,7 +25,6 @@ nav {
 			a {
 				text-decoration: none;
 				font-family: $base-font-name;
-				text-transform: uppercase;
```

Comment:
It's harder to read for dyslexic people, and screenreaders will read things like 'CONTACT US' like we're offering the opportunity to contact the United States.